docs/source/reference/kubernetes/examples/kueue-example.rst

```diff
@@ -41,30 +41,58 @@ Patch Kueue to support plain pods
-    kubectl -n kueue-system rollout status deployment kueue-controller-manager
+    .. tab-item:: Patch Kueue
+        :sync: patch-kueue-tab
+
+        .. code-block:: bash
+
+            # Extract and patch the config and save it to /tmp/kueueconfig.yaml
+            # This is required because SkyPilot creates and manages workloads as pods
+            kubectl -n kueue-system get cm kueue-manager-config -o jsonpath={.data.controller_manager_config\\.yaml} | yq '.integrations.frameworks += ["pod"]' > /tmp/kueueconfig.yaml
+            # Create an updated ConfigMap from /tmp/kueueconfig.yaml and apply the changes
+            kubectl -n kueue-system create cm kueue-manager-config --from_file=controller_manager_config.yaml=/tmp/kueueconfig.yaml --dry-run=client -o yaml | kubectl -n kueue-system apply -f -
+            # Restart the kueue-controller-manager pod with the following command
+            kubectl -n kueue-system rollout restart deployment kueue-controller-manager
+            # Wait for the restart to complete
+            kubectl -n kueue-system rollout status deployment kueue-controller-manager
+
+    .. tab-item:: Patch Kueue with GPUDirect enabled on GKE
+        :sync: patch-kueue-gke-tab
+
+        If you are using GKE, and want to enable the `GPUDirect networking stack <https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#required-features-capabilities>`_ for the GPU nodes, you need to patch the Kueue config to support plain pods and add ``networking.gke.io.networks`` to the ``resources.excludeResourcePrefixes`` to exclude the networking resources from the resource quota.
+
+        .. code-block:: bash
+            :emphasize-lines: 2
+
+            # Extract and patch the config and save it to /tmp/kueueconfig.yaml
+            kubectl -n kueue-system get cm kueue-manager-config -o jsonpath={.data.controller_manager_config\\.yaml} | yq '.integrations.frameworks += ["pod"] | .resources.excludeResourcePrefixes += ["networking.gke.io.networks"]' > /tmp/kueueconfig.yaml
+            # Create an updated ConfigMap from /tmp/kueueconfig.yaml and apply the changes
+            kubectl -n kueue-system create cm kueue-manager-config --from_file=controller_manager_config.yaml=/tmp/kueueconfig.yaml --dry-run=client -o yaml | kubectl -n kueue-system apply -f -
+            # Restart the kueue-controller-manager pod with the following command
+            kubectl -n kueue-system rollout restart deployment kueue-controller-manager
+            # Wait for the restart to complete
+            kubectl -n kueue-system rollout status deployment kueue-controller-manager
 
 Check that the patch is applied by running the following command:
 
 .. code-block:: bash
 
-    kubectl -n kueue-system get cm kueue-manager-config -o jsonpath={.data.controller_manager_config\\.yaml} | yq '.integrations.frameworks'
+    kubectl -n kueue-system get cm kueue-manager-config -o jsonpath={.data.controller_manager_config\\.yaml} | yq
```

Comment:
Will the removal of `'.integrations.frameworks'` output a lot of fields making it hard to find? May want to add an AND condition for `resources.excludeResourcePrefixes` if possible